### ISSUE TITLE
Slam and talents connected to it

### ISSUE THREAD

**Issue by Greywolf59** [WoWManiaUK/Redemption#3764]
**Links:**
Slam: https://www.wow-mania.com/armory/?spell=47475
Improved Slam: https://www.wow-mania.com/armory/?spell=12330
Juggernaut (talent): https://www.wow-mania.com/armory/?spell=12330
Juggernaut (aura debuff): https://www.wow-mania.com/armory/?spell=65156
Improved Thunder Clap: https://www.wow-mania.com/armory/?spell=12666
Bloodsurge: https://www.wow-mania.com/armory/?spell=46915
Incite (?): https://www.wow-mania.com/armory/?spell=50687

**What is Happening:**
Sorry guys, I did not know how exactly name this report as there is multiple issues.
So I will describe each talent one by one.
Improved Slam does not affect the Slam cast. When taking this talent the tooltip of slam cast changes to 0.5s bu real cast time is still 1.5s (visible on IceHud castbar).
Juggernaut does not seem to apply to slam critical chance. If I manage to pull crit with Slam during Juggernaut aura is active it is usually about third Slam, whereas before it was 1sr or 2nd use. Furthermore after slam or Mortal Strike crit this aura should be canceled, which does not happen after Slam crits. Aura is canceled only by critical strike of Mortal Strike.
Improved Thunder Clap is another talent that increaased Slam damage. I found about this only two days back and did not report it yet. It increases the slam damage about the 30% if specced 3/3 in Improved Thunder Clap. Now it does not increase the slam damage (which is good) but I do not think anyone actually tried to repair this part??
Bloodsurge: honorary mention as I do not play fury and I just know fury guys do have a problem with it.
Incite with a question mark. I do not know if it actually did raised the slam critical chance too, however now it seems the slam crits much less then before, whereas before this complications started it crited fairly often....I mean very often.
I also tested effects of Wrecking Crew https://www.wow-mania.com/armory/?spell=56614 but this talent does benefit the slam as it should.
Damage of slam does benefit properly (as far as I can say) from trinket and enchant procs.
I could not tested improved charge as the difference is rather small but my educated guess is it stopped benefiting the slam too.

Disclaimer: I did not test one and every talent of arms, fury or prot, as that is beyond my capabilities and ingame gold pool.

So to sum it up: Somehow many talents bugged or not that benefited slam got suddenly broke.

**What Should happen:**
Eh what should happen? Hard to tell :D. I guess slam should benefit properly from the talents

**Comment by Rushor** [comment]
i guess this was fixed here https://github.com/WoWManiaUK/Redemption/issues/3728

**Comment by Rushor** [comment]
if not we will reopen after update

**Comment by Greywolf59** [comment]
It is not a duplicate lol when two other spells got broke. Juggernaut and Improved Slam. That fix is talking about charge.
Do not rush it @Rushor

**Comment by Kitzunu** [comment]
@Dranah could you check please?

**Comment by Greywolf59** [comment]
So I logged PTR and there the slam works as it should. Aka 0.5s cast time when improved. Although I do not know how much this proves anything as my warr on PTR is like 2 months old if not more (old guild, old eq)
